### ISSUE TITLE
Android system navigation often uses light icons over white background in light mode

### ISSUE THREAD

**Issue by stephen304** [krawieck/lemmur#294]
### Device info

- OS: Android
- OS version: Lineage 18.1
- Device: Pixel 2
- Lemmur version: 0.6.0

### Describe the bug

The android system navigation buttons often use the wrong light/dark setting causing it to be hard to see. Toggling dark mode temporarily fixes the issue.

### Steps to reproduce

Simply changing apps seems enough to reproduce the issue

### Screenshots/Screencasts

https://user-images.githubusercontent.com/981915/140661116-25209cd8-9bc3-4afe-9a67-4e711139007a.mp4

**Comment by shilangyu** [comment]
Yeah, our implementation is pretty hacky, I'll check if we can make it more robust

**Comment by shilangyu** [comment]
Leaving it here to not forget: I need to check whether this solves it https://twitter.com/albert_wolszon/status/1464527087436537860?s=20
